fetch.bs

```diff
@@ -3779,6 +3940,79 @@ steps:
  <!-- This is really bad and needs to be handled differently at some point. -->
 </ol>
 
+<p>To <dfn>finalize timing info</dfn>, given <a for=/>fetch params</a> <var>fetchParams</var> and
+<a for=/>response</a> <var>response</var>, perform the following steps:
+<ol>
+ <li><p>Let <var>timingInfo</var> be <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
+
+ <li><p>Let <var>body</var> be <var>response</var>'s <a for=response>body</a>.
+
+ <li><p>Let <var>stream</var> be null.
+
+ <li><p>Let <var>flushAlgorithm</var> be an action that calls
+ <a>generate and report timing info</a> with <var>fetchParams</var> and <var>response</var>.
```

Comment:
I don't think this works. If you look at "fully read" or "incrementally read" those algorithms end up queuing a task (and there's also the processResponseEndOfBody callback which does a similar thing). This needs to be part of that task.

I think the way to get there is that we define fully read and incrementally read operations that take a response (rather than a response body) and those end up doing the extra thing of snapshotting the time and generating a report. That does mean we would have to store information on the response already, but it seems that by the time we create a response we should already know what information to hide based on TAO, right? And there's only a couple timing things still ongoing.

Reply:
Fixed in new version.